During EasyBuild, tool.setuptools_scm appears to rewrite '__version__' in _version.py
into 'version', leading to import issues.
Author: Guillaume Acke (UGent)
--- pyproject-orig.toml	2024-04-16 22:11:46.485575321 +0200
+++ pyproject.toml	2024-04-17 12:40:38.118879192 +0200
@@ -8,8 +8,8 @@
 
 [project]
 name = "qmflows"
+version = "1.0.0"
 dynamic = [
-    "version",
     "readme",
 ]
 description = "Automation of computations in quantum chemistry."
@@ -101,8 +101,8 @@
 [tool.setuptools.dynamic]
 readme = { file = ["README.rst"], content-type = "text/x-rst" }
 
-[tool.setuptools_scm]
-write_to = "src/qmflows/_version.py"
+#[tool.setuptools_scm]
+#write_to = "src/qmflows/_version.py"
 
 [tool.mypy]
 plugins = "numpy.typing.mypy_plugin"
